ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ConnectServerErrors.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2018 Confluent Inc.
```

Comment:
nit: year, and also for the other newly added file.

Reply:
Fixed in https://github.com/confluentinc/ksql/pull/8480/commits/7ac1fe2c1b271aec0498c7f1315b34f0681bb7a4#diff-583e52f2f5ace65057d66ec857d4f304326c56d5b542d0c14c2e685c97c875c0R2, https://github.com/confluentinc/ksql/pull/8480/commits/7ac1fe2c1b271aec0498c7f1315b34f0681bb7a4, https://github.com/confluentinc/ksql/pull/8480/commits/7ac1fe2c1b271aec0498c7f1315b34f0681bb7a4#diff-d412eea1cfe4a271bb0654c2a543c2ad7d7cb53febd8e522c9c04dcad5f22169R2.